src/aleph/vm/models.py

```diff
@@ -64,7 +67,7 @@ class VmExecution:
     original: ExecutableContent
     message: ExecutableContent
     resources: Optional[AlephFirecrackerResources] = None
-    vm: Optional[AlephFirecrackerExecutable] = None
+    vm: Optional[AlephFirecrackerExecutable | AlephQemuInstance] = None
```

Comment:
The `|` syntax is not supported for Python versions < 3.10, but we still support 3.9: https://github.com/aleph-im/aleph-vm/blob/main/pyproject.toml#L96

Reply:
good catch, fixed it